test/Atmos/Model/driver_spacing_test.jl

```diff
@@ -0,0 +1,40 @@
+using MPI
+using ArgParse
+using ClimateMachine
+ClimateMachine.init()
+using ClimateMachine.MPIStateArrays
+using ClimateMachine.Atmos
+using ClimateMachine.ConfigTypes
+using ClimateMachine.GenericCallbacks
+using ClimateMachine.DGMethods
+using ClimateMachine.Mesh.Grids
+using ClimateMachine.Mesh.Geometry
+using ClimateMachine.Orientations
+
+using Test
+
+@testset "EmptyModel Get Nodal Distance GCM" begin
```

Comment:
Based on the same question I had above, should this really be printing `"EmptyModel Get Element Size GCM"`or `"EmptyModel Get Grid Size GCM"` (if I am not misunderstanding the code)? Thanks